stubs/objgraph/METADATA.toml

```diff
@@ -0,0 +1,6 @@
+version = "3.6.*"
+upstream_repository = "https://github.com/mgedmin/objgraph"
+requires = ["graphviz"]
+
+[tool.stubtest]
+apt_dependencies = ["graphviz"]
```

Comment:
I don't think there's any point adding `graphviz` as a dependency here, since `graphviz` doesn't have a `py.typed` file and you don't actually import anything from it in these stubs.

I'm also not sure it needs to be listed as something we need to `apt-install` for stubtest, either? Stubtest passes fine for me on this PR locally without installing `graphviz`. I suggest removing all references to `graphviz` here, which should make the stub-uploader tests pass:

```suggestion
```